app/page/template/content/conversation.htm

```diff
@@ -1,16 +1,25 @@
-<div id="conversation" class="conversation" data-bind="drop_file: $root.inputBar.onDropFiles">
+<style>
+.conversation-loading {
+ display: none;
+}
+.loading .conversation-loading {
+  display: block;
+}
+.loading .conversation-title-bar,
+.loading .message-list,
+.loading .conversation-input-bar {
+  opacity: 0;
+}
+</style>
+<div id="conversation" class="conversation" data-bind="drop_file: $root.inputBar.onDropFiles, css: { loading: !$root.messageList.conversationLoaded() }">
```

Comment:
@AndyLnd that is where I need you. What would be a perfect DOM structure for you?